src/components/standup/standupContainer.module.scss

```diff
@@ -0,0 +1,152 @@
+$theme-CornFlowerBlue: #1c64f2;
+$theme-Biscay: #1c325e;
+$theme-white: #ffffff;
+$theme-FrenchRose: #f1416c;
+$theme-FunBlue: #1e429f;
+$theme-CeriseRed: #d61f69;
+$theme-PaleSky: #6b7280;
+$theme-Mischka: #d1d5db;
+$theme-AthensGray: #e5e7eb;
+$radius: 0.5rem;
+
+.standupContainer {
+    display: flex;
+    flex-direction: column;
+    margin-top: 1.875rem;
+    width: 100%;
+    align-items: center;
+    justify-content: center;
+}
+
+.standupBanner {
+    background-color: $theme-Biscay;
+    width: 83.7%;
+    height: 12%;
+    color: $theme-white;
+    padding: 1.5rem 0 1.5rem 3.0625rem;
+    border-radius: $radius;
+}
+
+.standupBanner p {
+    font-family: 'Poppins', sans-serif;
+    font-size: 1.25rem;
+    font-weight: 600;
+}
+
+.standupBanner p span {
+    color: $theme-FrenchRose;
+    text-decoration: underline;
+    margin: 0 0.25rem;
+}
+
+.standupUpdateContainer {
+    width: 50.7%;
+    margin-top: 3.125rem;
+}
+
+.standupForm {
+    margin-top: 2.1875rem;
+}
+
+.standupUpdateContainer h1 {
+    color: $theme-FunBlue;
+    font-family: 'Inter', sans-serif;
+    font-size: 2.2rem;
+    padding-left: 1rem;
+}
+
+.formFields {
+    margin-bottom: 1.25rem;
+    border: none;
+}
+
+.updateHeading {
+    color: $theme-PaleSky;
+    font-size: 1.4rem;
+    font-family: 'Inter', sans-serif;
+}
+
+.inputField {
+    height: 58px;
```

Comment:
As per design guidelines i never saw 58 px to be used.

Reply:
![image](https://github.com/Real-Dev-Squad/website-status/assets/82165483/facb60f1-88a8-4538-9b98-fdc54d7a8d70)

I have used calculation for it as 58px/4px = 14.5units so 58px/16px = 3.625rem(approx 4rem)

Reply:
fixed